helix-view/src/clipboard.rs

```diff
@@ -120,6 +124,27 @@ mod provider {
         }
     }
 
+    #[cfg(target_os = "windows")]
+    #[derive(Debug)]
+    pub struct WindowsProvider;
+
+    #[cfg(target_os = "windows")]
+    impl ClipboardProvider for WindowsProvider {
+        fn name(&self) -> Cow<str> {
+            Cow::Borrowed("Windows Clipboard")
```

Comment:
```suggestion
            Cow::Borrowed("windows")
```
Let's follow the same casing.

Reply:
If we use `clipboard-win` crate, I would go with `clipboard-win` (if we were using winapi I would have suggested `winapi`).